docs/1-benchpark-setup.rst

```diff
@@ -0,0 +1,27 @@
+=========
+Benchpark setup
+=========
+
+Python 3.6 and git are required to check out and install Benchpark::
+
+  git clone git@github.com:LLNL/benchpark.git $benchpark  
+  benchpark setup [--spack=spack_dir] [--ramble=ramble_dir]
+
+Benchpark uses the following open source projects for specifying configurations:
+
+* `Ramble <https://github.com/GoogleCloudPlatform/ramble>`_ to specify run configurations
+* `Spack <https://github.com/spack/spack>`_ to specify build configurations
+
+If you already have Spack and/or Ramble on your system and you want to use these
+installs, simply point Benchpark at these installs by passing in their locations.
+
+
+What the ``benchpark setup`` script does for you
```

Comment:
`benchpark` is a script, but `benchpark setup` is a subcommand, and IMO referring to it as a script is confusing; I think it would be clearer to say:

> What ``benchpark setup`` does for you